### PR TITLE
More mutiny to mutiny cleanup

### LINKED ISSUE #262: should revert this back to trace
should revert this back to trace

_Originally posted by @benthecarman in https://github.com/BitcoinDevShop/mutiny-web-poc/pull/207#discussion_r1058663425_

### PR DESCRIPTION
**Pull request by TonyGiorgio**
Handles the rest of the PR feedback in https://github.com/BitcoinDevShop/mutiny-web-poc/pull/144

- Fixes #262

### CODE REVIEW

File: node-manager/src/node.rs [hunk 1]
```diff
@@ -108,7 +108,7 @@ impl PubkeyConnectionInfo {
                 .context("connect_peer requires peer connection info")?;
         };
 
-        if connection.starts_with("mutiny:") {
+        if connection.to_lowercase().starts_with("mutiny:") {
             let stripped_connection = connection.strip_prefix("mutiny:").expect("should strip");
```
Comment (benthecarman) [inline node-manager/src/node.rs:112]: I think this strip_prefix will fail if it isn't all lowercase now

File: node-manager/src/socket.rs [hunk 2]
```diff
@@ -224,10 +234,10 @@ impl MultiWsSocketDescriptor {
                             };
                         }
                         Message::Bytes(msg) => {
-                            debug!("received a binary message on multi socket...");
                             // This is a mutiny to mutiny connection with pubkey + bytes
                             // as the binary message. Parse the msg and see which pubkey
                             // it belongs to.
+                            debug!("received a binary message on multi socket...");
```
Comment (benthecarman) [inline node-manager/src/socket.rs:240]: should this be trace?

**Comment by benthecarman** [comment]
Looks like CI is throwing some clippy errors now

File: node-manager/src/node.rs (hunk 1 above)
Comment (TonyGiorgio) [reply node-manager/src/node.rs]: good catch, doing it at the beginning now so it should apply to both

File: node-manager/src/socket.rs (hunk 2 above)
Comment (TonyGiorgio) [reply node-manager/src/socket.rs]: moved to trace

**Review by benthecarman — APPROVED** [review]

**Comment by TonyGiorgio** [comment]
Tested the vercel deployment and it worked well for me.

**Pull request merged**